app/scripts/Xprize-VisualRetargeting.xml

```diff
@@ -95,7 +70,7 @@
   <connection>
     <from>/idyntree-yarp-visualizer/image</from>
     <to>/joypadDevice/Oculus/gui_0</to>
-    <protocol>mjpeg</protocol>
+    <protocol>fast_tcp</protocol>
```

Comment:
Why we are changing it? If not wrong, `mjpeg` doing some compression of the data before buffering it to the network. Normally, for images, `mjpeg`is recommended to be used.